contrib/cirrus/packer/ubuntu_packaging.sh

```diff
@@ -144,22 +150,23 @@ $BIGTO ooe.sh $SUDOAPTGET upgrade
 echo "Installing general testing and system dependencies"
 # Necessary to update cache of newly added repos
 $LILTO ooe.sh $SUDOAPTGET update
-$BIGTO ooe.sh $SUDOAPTGET install ${INSTALL_PACKAGES[@]}
+$BIGTO ooe.sh $SUDOAPTGET install "${INSTALL_PACKAGES[@]}"
 
 if [[ ${#DOWNLOAD_PACKAGES[@]} -gt 0 ]]; then
     echo "Downloading packages for optional installation at runtime, as needed."
     $SUDO ln -s /var/cache/apt/archives "$PACKAGE_DOWNLOAD_DIR"
-    $LILTO ooe.sh $SUDOAPTGET install --download-only ${DOWNLOAD_PACKAGES[@]}
-    ls -la "$PACKAGE_DOWNLOAD_DIR/"
+    $LILTO ooe.sh $SUDOAPTGET install --download-only "${DOWNLOAD_PACKAGES[@]}"
 fi
 
-echo "Installing runtime tooling"
-# Save some runtime by having these already available
+echo "Configuring Go environment"
+$SUDO ln -s /usr/lib/go-1.14/bin/go /usr/bin/go
```

Comment:
This is a little scary, can you explain why it's needed? Is there another way to do it? And if not, would it be a good idea to use `ln -sf` in case there's already a `/usr/bin/go`?

Reply:
Sadly, there are multiple versions of golang available in Ubuntu for each release and this is how they're suppose to be installed system-wide.  Otherwise using them requires some (more fugly) env. var. monkeying.  Though the -sf isn't a bad idea, would you be okay with me adding the `f` but not re-rolling the VM images?  I can also add a comment to that line.

Reply:
Oh heck yes. None of my comments (except the docker one, which I think is obsolete knowledge) require re-spinning CI:IMG. A comment would be appreciated.

Reply:
okay, done.